### ISSUE TITLE
Safari Side Bar Issue

### ISSUE THREAD

**Issue by coderinblack08** [benawad/dogehouse#2536]
Go on Safari and open DogeHouse. The left-hand panel is messed up big time.

![Screen Shot 2021-05-03 at 11 45 35 AM](https://user-images.githubusercontent.com/41022901/116918695-1611a080-ac05-11eb-97e2-bbbc3460a641.png)

**Comment by ajmnz** [comment]
<img width="1423" alt="Screenshot 2021-05-04 at 13 43 35" src="https://user-images.githubusercontent.com/57961822/116998573-ba1b4a80-acde-11eb-9d6c-ce0516c09dc6.png">

Looks good to me, upcoming rooms are messed up though

**Comment by coderinblack08** [comment]
Are you on Big Sur? It's funky for me... I disabled all my extensions too

**Comment by ajmnz** [comment]
> Are you on Big Sur? It's funky for me... I disabled all my extensions too

yep I am, can you open devtools and see what rule isn’t working? I don’t remember if that uses flex gap, but if it does we’ll switch it to margin
